docs/user/library/coverage/geotiff.rst

```diff
@@ -5,7 +5,6 @@ The gt-geotiff module in the plugin group provides access to the GeoTiff file fo
 
 **References**
 
-* http://www.jroller.com/jsight/entry/back_to_code_geotools_geotiff
```

Comment:
Since this URL now gives a 404, removing it is reasonable.
However, I wonder if there is a better option?
@aaime or @jodygarnett do you know of any more recent reference material for GeoTools / GeoTiff?